server/image_processor.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+from PIL import Image
+
+size = 35, 35
+angular_segments_count = 120
+angular_segments = 360 / angular_segments_count
+
+
+def process_image_for_mcu(image_file):
+    image = Image.open(image_file)
+    image.thumbnail(size)
+
+    output_pixels = []
+    for j in range(size[0]):
+        output_pixels.append([])
+
+    for theta in range(angular_segments_count):
+        copied_image = image.copy()
+        rotated_image = copied_image.rotate((360/angular_segments_count) * theta)
+        pixels = list(rotated_image.getdata())[(35 * 17):(35 * 17) + 35]
```

Comment:
Are these 35s supposed to be pulled from `size`?

Reply:
Ohh yeah this is my haste and partially due to me not being able to coerce python into doing what I wanted with a float <=> integer number auto-casting. I'll extract these magic numbers into named constants.

Reply:
Also, just so this is documented somewhere the overall rationale here is:

```
0-indexed 2D color matrix of a 35x35 image 
========================================
| (0, 0)  . . . (0, 17) . . .  (0, 34) |
|    .              .            .     |
|    .              .            .     |
|    .              .            .     |
| (17, 0) . . . (17, 17) . . . (17, 34)|
|    .              .             .    |
|    .              .             .    |
|    .              .             .    |
| (34, 0) . . . (34, 17) . . . (34, 34)|
========================================

- 17 is significant as `17 = floor(37/2)`
```

We're trying to extract the range of pixels in the middle from `(17, 0)` to `(17, 17)`. When the row-major 2D matrix is flattened to a list, these pixels should be in the range indexed by `(17,0) => 35*17 + 0` to `(17,7) => 35*17 + 17`. 

If we extract this range each time the image is rotated 3 degrees, then we should have a _nice_ contiguous view of the image through it's entire rotation.